docs/usage.rst

```diff
@@ -257,6 +257,14 @@ each scenario instead of flushing the entire database. Tests run much quicker,
 however HTTP server is not started and therefore web browser automation is
 not available.
 
+``--behave-runner``
+*******************
+
+Allows the modification of the underlining runner class that Behave uses by
+default. This will allow you to customize where your feature files are located.
```

Comment:
```
Allows to override the test runner class that Behave uses by default.
```

Is "where your feature files are located" just an example use case? Then please insert a `, e.g., ` in the second sentence.

Reply:
Yup. Very clear, done.